scripts/ci_setup.py

```diff
@@ -378,6 +379,15 @@ def main(args: Any):
     else:
         dotnet.setup_dotnet(args.dotnet_path)
 
+    framework = ChannelMap.get_target_framework_moniker(args.channel)
+    if framework == 'net8.0' or framework == 'nativeaot8.0':
+        # Copy the global.json file to global.bak.json
```

Comment:
This comment and the copy below do not match - _global.bak.json_ vs _global.net9.json_.

Reply:
Great catch, updated.